### ISSUE TITLE
Public release 3.0 bad resolution jumping.

### ISSUE THREAD

**Issue by retrobenny** [gonetz/GLideN64#1713]
On PJ64 2.4.0-455 it keeps changing resolution at random constantly and it causes a multitude of problems such as black screen flickers and jumpy rendering when not blacked out.

Beetle Adventure Racing is a good example of this,I noticed it in a few other games but forget which.
Kinda hard to get it in picture form due to the type of bug it is.
It may be caused by setting the resolution when a game is already running but that is a really irritating reason if true.
When the issue happened,there was a noticeable gradient on the sky-box.

Interpreter seems to make it act up from the looks of it.
Ending and starting emulation seems to also help cause it.

Another game with problems even worse indefinitely is Air Boarder 64 when I had another go at attempting to get a pacing code for proper 60fps only to find it missing the Human logo and seizure flickering once the intro title screen starts.

**Comment by gonetz** [comment]
I use PJ64 2,3 and I never had resolution jumping.
I need bug scenario, which I can reproduce.

**Comment by retrobenny** [comment]
Hmm,maybe check Airboarder 64 since that has its own issue of seizure flickering for me.

Maybe I should see what happens if I force my Intel GPU to do the work or vice versa if I messed up settings.

**Comment by gonetz** [comment]
I need screen shot or shot video demonstrating what is wrong.
Heisenbugs have near-zero chance to be fixed.

**Comment by retrobenny** [comment]
Sorry for the delay but here is a video I already had recorded of the Airboarder 64 seizure flickering which happens on PJ64 2.4.0-455 as seen in the video.

[2018-01-30 15-38-43.zip](https://github.com/gonetz/GLideN64/files/1804737/2018-01-30.15-38-43.zip)

**Comment by gonetz** [comment]
Yes, Air Boarder (J) flickers badly. It is not Public release 3.0 regression - Release 2.0 works the same.
Air Boarder (E) works ok.

**Comment by retrobenny** [comment]
So GLideN64 still doesn't like (J) region games all that much for some titles.
With (E) being 50hz,I was utilizing the (J) region to attempt a 60fps code since a long while back,and to see it break with flickering ruins my ability to search for codes on it using the debugger.
